src/node.ts

```diff
@@ -404,13 +404,26 @@ const generatePreviewKey = () => {
 	return previewKey;
 };
 
-const getDynamicPageURL = async ({ contentID, preview, slug }) => {
+const getDynamicPageURL = async ({ contentID, preview, slug, locale }:{
```

Comment:
Can we take the opportunity to make this a strongly typed interface for these params?  The locale can be an optional string.